src/app/(customerFacing)/layout.tsx

```diff
@@ -35,8 +38,11 @@ console.log(cartItems?.length, "hello")
             <UserNavbar />
             <NavLink href="/cart">
               
-            
-              <ShoppingCart />
+            <div className="flex relative">    
+            <ShoppingCart />
+            <div className="absolute top-0 right-0 translate-x-1/2 -translate-y-1/2  text-black text-xs rounded-full px-1"> {cartItems?.length}</div>
+            </div>
```

Comment:
Wonderful!

Let's extract these lines into it's own component, then import it here. 

That way we don't need to make the entire layout page a client component.